sdk/eventhub/Microsoft.Azure.EventHubs/tests/Client/WebSocketTests.cs

```diff
@@ -13,89 +16,109 @@ namespace Microsoft.Azure.EventHubs.Tests.Client
+                Assert.True(eventHubRuntimeInformation.PartitionIds != null, "eventHubRuntimeInformation.PartitionIds was null!");
+                Assert.True(eventHubRuntimeInformation.PartitionIds.Length != 0, "eventHubRuntimeInformation.PartitionIds.Length was 0!");
+
+                TestUtility.Log("Found partitions:");
+                foreach (string partitionId in eventHubRuntimeInformation.PartitionIds)
+                {
+                    TestUtility.Log(partitionId);
+                }
             }
         }
 
+
+
         [Fact]
         [LiveTest]
         [DisplayTestMethodName]
         public async Task SendAndReceive()
         {
             string targetPartitionId = "0";
 
-            // Create new client with updated connection string.
             TestUtility.Log("Creating Event Hub client");
-            var ehClient = EventHubClient.CreateFromConnectionString(webSocketConnString);
-
-            PartitionSender sender = null;
-            try
-            {
-                // Send single message
-                TestUtility.Log("Sending single event");
-                sender = ehClient.CreatePartitionSender(targetPartitionId);
-                var eventData = new EventData(Encoding.UTF8.GetBytes("This event will be transported via web-sockets"));
-                await sender.SendAsync(eventData);
-            }
-            finally
+            await using (var scope = await EventHubScope.CreateAsync(2))
             {
-                await sender?.CloseAsync();
-            }
 
-            PartitionReceiver receiver = null;
-            try
-            {
-                // Receive single message.
-                TestUtility.Log("Receiving single event");
-                receiver = ehClient.CreateReceiver(PartitionReceiver.DefaultConsumerGroupName, targetPartitionId, EventPosition.FromStart());
-                var msg = await receiver.ReceiveAsync(1);
-                Assert.True(msg != null, $"Failed to receive single event from partition {targetPartitionId}");
-            }
-            finally
-            {
-                await receiver?.CloseAsync();
+                var connectionString = TestUtility.BuildEventHubsConnectionString(scope.EventHubName);
+                var ehClient = EventHubClient.CreateFromConnectionString(GetWebSocketConnectionString(connectionString));
+
+                PartitionSender sender = null;
+                try
+                {
+                    // Send single message
+                    TestUtility.Log("Sending single event");
+                    sender = ehClient.CreatePartitionSender(targetPartitionId);
+                    var eventData = new EventData(Encoding.UTF8.GetBytes("This event will be transported via web-sockets"));
+                    await sender.SendAsync(eventData);
+                }
+                finally
+                {
+                    await sender?.CloseAsync();
+                }
+
```

Comment:
nit:  extra blank lines that we should remove.

Reply:
Removed extra blank for all files we updated.